tedge/src/system_services/error.rs

```diff
@@ -17,6 +17,9 @@ pub enum SystemServiceError {
 
     #[error("Unsupported operation.")]
     UnsupportedOperation,
+
+    #[error("Service Manager: '{0}' is not available on the system or elevated permissions have not been granted.")]
```

Comment:
Is not it this error message specific to the WSL?. Because elevated permission is more of a windows thing right?

Reply:
This error may occur on any OS which doesn't have supported service manager installed.
Elevated permissions or elevated rights applies to pretty much all OSes.